app/webpacker/styles/govspeak/_highlights.scss

```diff
@@ -4,5 +4,9 @@
     border-radius: 9px;
     padding: 40px;
     margin: 50px;
+    iframe {
+      width: 444px;
+      height: 269px;
+    }
```

Comment:
Does that override the default values we set in govspeak? So the videos in information blocks are smaller than the regular ones?

Reply:
It doesn't look too. If you go to `/ambition/year-1/autumn-1/topic-2/part-1` the video is the size set in govspeak - the old size as is the case in this review app